source/adapters/level_zero/adapter.hpp

```diff
@@ -10,10 +10,14 @@
 
 #include <atomic>
 #include <mutex>
+#include "logger/ur_logger.hpp"
 
 struct ur_adapter_handle_t_ {
   std::atomic<uint32_t> RefCount = 0;
   std::mutex Mutex;
+  logger::Logger logger = logger::create_logger("level_zero");
 };
 
 extern ur_adapter_handle_t_ Adapter;
+
+
```

Comment:
redundant extra empty lines

Reply:
you can use `make cppformat` command to remove stuff like that automatically